blazor/bullet-chart/events.md

```diff
@@ -160,4 +160,47 @@ The [LegendRender](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Chart
         // Here, you can customize the code.
     }
 }
+
+```
+
+The [PointerClick](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.BulletChartEvents.html#Syncfusion_Blazor_Charts_BulletChartEvents_PointerClick) event triggers before each legend item rendering.
```

Comment:
Why explaining legend item rendering here?
`The [PointerClick](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.BulletChartEvents.html#Syncfusion_Blazor_Charts_BulletChartEvents_PointerClick) event triggers before each legend item rendering.`

Reply:
Description changed